src/main/java/racingGame/Business/Process.java

```diff
@@ -0,0 +1,62 @@
+package racingGame.Business;
+
+import java.util.ArrayList;
+import racingGame.ResultView;
+import racingGame.model.Car;
+import racingGame.util.RandomUtil;
+
+public class Process {
+    
+    public static final int CAR_FORWARD_CRITERIA = 3;
+    private final ArrayList<Car> cars = new ArrayList<>();
+    private int moves;
+    
+    public void run(int cars, int moves){
+        init(cars, moves);
+        for(int i = 0; i < moves; i++) {
+            nextMove();
+            ResultView.moveDonePrint();
+        }
+    }
+    
+    private void nextMove() {
+        for(Car car: cars) {
+            ResultView.forwardCarPrint(car.findLocation());
+            if(RandomUtil.generateInt() > CAR_FORWARD_CRITERIA) {
+                car.move();
+            }
+        }
+    }
+    
+    public ArrayList<Car> joinCars() {
```

Comment:
```suggestion
    public List<Car> joinCars() {
```
내부 구현체가 LinkedList와 같은 구현체로 변경하더라도 외부에 변경이 발생하지 않음. 따라서 인터페이스가 있다면 인터페이스를 외부에 노출할 것을 추천

Reply:
DIP 원칙을 잊고있었네요 짚어주셔서 감사드립니다!